cmd/schemagen/map_types.go

```diff
@@ -88,18 +85,3 @@ func mapScyllaToGoType(s string) string {
 
 	return camelize(s) + "UserType"
 }
-
-func typeToString(t interface{}) string {
-	tType := fmt.Sprintf("%T", t)
-	switch tType {
-	case "gocql.NativeType":
-		return t.(gocql.NativeType).String()
-	case "gocql.CollectionType":
-		collectionType := t.(gocql.CollectionType).String()
-		collectionType = strings.Replace(collectionType, "(", "<", -1)
-		collectionType = strings.Replace(collectionType, ")", ">", -1)
-		return collectionType
-	default:
-		panic(fmt.Sprintf("Did not expect %v type in user defined type", tType))
-	}
```

Comment:
why remove this function?

Reply:
There is difference between `KeyspaceMetadata` in  [scylladb/gocql](https://github.com/scylladb/gocql/blob/c570ec02e0641060c3ebedfe09703977dd6ed8b3/metadata.go#L48-L59) and [gocql/gocql](https://github.com/gocql/gocql/blob/c570ec02e0641060c3ebedfe09703977dd6ed8b3/metadata.go#L48-L59)

On `scylladb/gocql` attribute `Type` is string, a raw name of the type, while on `gocql/gocql` it is `TypeInfo`.
So, this function converts `TypeInfo` to a string, a raw name of the type.
Which is why this function is not needed anymore